virtualization/windowscontainers/quick_start/quick_start_windows_10.md

```diff
@@ -22,7 +22,8 @@ This quick start is specific to Hyper-V containers on Windows 10. Additional qui
 - One physical computer system running Windows 10 Anniversary Edition (Professional or Enterprise).   
 - This quick start can be run on a Windows 10 virtual machine however nested virtualization will need to be enabled. More information can be found in the [Nested Virtualization Guide](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/user_guide/nesting).
 
-> Critical updates are needed in order for the Windows Container feature to function. Please install all updates before working through this tutorial.
+> Critical updates are needed in order for the Windows Container feature to function. Please install all updates before proceeding.
+> To check your OS version, run winver.exe, and compare the version shown to [Windows 10 update history](https://support.microsoft.com/en-us/help/12387/windows-10-update-history). Make sure you have 14393.222 or later. This will also avoid a issue between Hyper-V and Bitlocker that will make a system unbootable.
 
```

Comment:
Suggestion:
"You must install critical updates for Windows Containers to run. To check your OS version..." Or replace "run" with "work". 
I'd delete this last sentence "This will also..." We're already telling them it's a critical update. If you think we have to have that info in, then maybe this instead "Make sure you have 14303.22 or a later version.  If you don't upgrade to the latest version, your Windows Container may be unbootable because of an issue between Hyper-V and Bitlocker."